CONTRIBUTING.md

```diff
@@ -13,7 +13,10 @@ If you need to make asset changes:
 * `composer install`
 * `npm install`
 * Add the missing values from `.env` to a `.env.local` file
-  * Use `https://vision.googleapis.com/` as the `APP_GOOGLE_CLOUD_VISION_ENDPOINT`, with your own [Cloud Vision API](https://cloud.google.com/vision) key as the `APP_GOOGLE_CLOUD_VISION_KEY`. Google gives you 1,000 free lookups per month.
+  * Create a new Google service account at https://console.cloud.google.com/iam-admin/serviceaccounts Google gives you 1,000 free lookups per month.
+  * Give the service account the *Compute Engine Service Account* role.
+  * Add a new key for the service account (of type JSON) and save it somewhere.
```

Comment:
I'm assuming tests/fixtures/google-account-keyfile.json is an example of this. I wasn't sure where to get the `private_key_id`, and whether the `client_email` and `client_id` are required. I tried to guess my way through it and am getting the error `Warning: openssl_sign(): supplied key param cannot be coerced into a private key` when attempting to transcribe with Google Cloud Vision. Is there documentation on the structure of this keyfile somewhere?

When I started from scratch with a new service account, it seems the 'Compute Engine Service Account' role (and most all of the roles) are already "enabled". Is that expected?

Reply:
Sorry, good point. The entire JSON file is actually provided in the Google UI, and it doesn't need to be edited. I'll update the instructions to make this clearer.

Reply:
> When I started from scratch with a new service account, it seems the 'Compute Engine Service Account' role (and most all of the roles) are already "enabled". Is that expected?

I'm not really sure. I had to select them, so maybe there are different defaults for different projects or something? I must admit, I'm not totally across how all of the IAM stuff works.